spec/directions/bake_chapter_summary_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'spec_helper'
+
+RSpec.describe Kitchen::Directions::BakeChapterSummary do
+
+  let(:chapter) do
+    chapter_element(
+      <<~HTML
+        <div data-type='page' id ="01">
+          <h1 data-type="document-title" itemprop="name">First Title</h1>
+          <section class="summary" data-element-type="section-summary">
+              <h3 data-type='title'>Summary</h3>
+              <p>Many paragraphs provide a good summary.</p>
+          </section>
+        </div>
+        <div data-type='page' id="02">
+          <h1 data-type="document-title" itemprop="name">Second Title</h1>
+          <section class="summary" data-element-type="section-summary">
+              <h3 data-type='title'>Summary</h3>
+              <p>Ooh, it's another bit of text.</p>
+          </section>
+        </div>
+      HTML
+    )
+  end
+
+  #TODO: check that it:
+  #Replaces title with h3 title (children: os-number, os-divider, os-text)
+  #
+
+  context 'when v1 is called on a chapter' do
+    it 'works' do
+      described_class.v1(chapter: chapter, metadata_source: metadata_element)
+      expect(
+        chapter
+      ).to match_normalized_html(
+        <<~HTML
+          <div data-type="chapter">
+            <div data-type="page" id="01">
+              <h1 data-type="document-title" itemprop="name">First Title</h1>
+            </div>
+            <div data-type="page" id="02">
+              <h1 data-type="document-title" itemprop="name">Second Title</h1>
+            </div>
+            <div class="os-eoc os-summary-container" data-type="composite-page" data-uuid-key=".summary">
+              <h2 data-type="document-title">
+                <span class="os-text">Summary</span>
```

Comment:
This "Summary" text is pulled from `lib/locales/en.yml`, a set of English translations of certain words.  For our Spanish books, we'll have an `es.yml` or other file that has "Summary" translated into Spanish.  All words that we insert into books should be in these "locales" files for internationalization (aka "i18n" == "i followed by 18 letters followed by n" == "internationalization").

One day we may decide to change the English word we use here but we don't want this spec to break because of it.  To protect ourselves against that, we tell rspec to use a fake translation for this term with this block at the top of this file:

```ruby
before do
  stub_locales({
    'eoc_summary_title': 'Summary'
  })
end
```

You can see this in action [here in another spec](https://github.com/openstax/kitchen/blob/master/spec/directions/bake_numbered_table/v1_spec.rb#L5-L9).

Without this, if you change the value for the `eoc_summary_title` in `en.yml`, this test would break.  With this, such changes do not break the spec.

Read a tiny bit more about i18n [in the README](https://github.com/openstax/kitchen#internationalization-i18n).